### ISSUE TITLE
UserAvatar user initials doesn't pass contrast checks

### ISSUE THREAD

**Issue by davinotdavid** [thunderbird/services-ui#146]
**Describe the bug**
The UserAvatar's user initials fails contrast checks. I believe this is due to the semi-transparent background and using it against a dark NavBar.

**To Reproduce**

On a service that has `services-ui` installed v1.0.2+

1. Render the `UserAvatar` component on a dark background
2. Note that the initial is too dark / fails contrast checks

**Expected behavior**
- Light / Dark modes are applied successfully
- Both modes have user initials readable regardless of background

**Actual behavior**
- Light / Dark modes are the same
- User initials are not readable

**Screenshots**

Dark mode
<img width="225" height="179" alt="Image" src="https://github.com/user-attachments/assets/cbf058f8-87d7-4940-b816-dd287c4a6585" />

Light mode
<img width="235" height="175" alt="Image" src="https://github.com/user-attachments/assets/c0b1096a-1f6c-4aef-96a3-0a834c868418" />
